app/controllers/api/v1/revenue_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::V1::RevenueController < ApplicationController
+  def index
```

Comment:
decided to organize my controllers based on return values, since these two methods are returning revenue it makes sense for them to be together